templates/taxi/driver_list.html

```diff
@@ -1,34 +1,39 @@
 {% extends "base.html" %}
+{% load crispy_forms_filters %}
 
 {% block content %}
-    <h1>
-      Driver List
-      <a href="{% url 'taxi:driver-create' %}" class="btn btn-primary link-to-page">
-        Create
-      </a>
-    </h1>
+  <h1>
+    Driver List
+    <a href="{% url 'taxi:driver-create' %}" class="btn btn-primary link-to-page">
+      Create
+    </a>
+  </h1>
+  <form method="get" action="" class="form-inline">
+    {{ search_form|crispy }}
+    <input type="submit" value="Submit" class="btn-btn-primary">
```

Comment:
There is a typo in the class attribute value for the submit button. It should be 'btn btn-primary' instead of 'btn-btn-primary'.